components/dial/stories/dial.stories.js

```diff
@@ -62,25 +63,71 @@ export default {
 	},
 };
 
+// the !auto-docs tag removes the component from the auto-template
 export const Default = Template.bind();
 Default.args = {};
+Default.tags = ['!autodocs'];
```

Comment:
I found `!autodocs` here: https://storybook.js.org/docs/writing-docs/autodocs and I'm using it to remove this Default dial story from our auto-template, so that I can just display the dial groups like on the [docs site](https://opensource.adobe.com/spectrum-css/dial.html)